perfkitbenchmarker/pkb.py

```diff
@@ -491,4 +504,10 @@ def Main(argv=sys.argv):
         '%s\nUsage: %s ARGS\n%s', e, sys.argv[0], FLAGS)
     sys.exit(1)
 
+  try:
+    SetUpPKB()
+  except Exception as e:
+    logging.error('Got exception %s', e)
+    sys.exit(1)
```

Comment:
I don't think this try/except is necessary. If an exception is raised, python will exit with a non-zero status.